src/core/include/openvino/op/util/evaluate_helpers.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/partial_shape.hpp"
+#include "openvino/runtime/tensor.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+
+/**
+ * @brief Get the tensors shapes as ov::PartialShape.
+ *
+ * @param tensors  Input tensors vector to get its shapes.
+ * @return Vector of partial shapes sam size as input tensor vector.
+ */
+std::vector<PartialShape> get_tensors_partial_shapes(const TensorVector& tensors);
```

Comment:
I think it should be a part of dev API. 
Moreover you need to export this method if you want to have it as a part of public API.

Reply:
We need to export public API from the library. Because in other case users won't be able to use these methods. API should be consistant.